desktop/apps/editorial_features/components/gucci/components/partner/__tests__/partner_block.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { shallow } from 'enzyme'
```

Comment:
Yesss -- have we used Enzyme in force before? If not, great addition 👍

Reply:
One suggestion I would make is to always try to use `mount` as it better reflects whats going on in the DOM in a real-world environment. The speed difference is negligible.